First Contribution: I added a circle to the confetti, I think

JHMuir opened the pull request.

JHMuir committed:
Hi did I do it right?

EthanThatOneKid commented on static/assets/1st/circle-thing.svg:
```diff
@@ -0,0 +1 @@
+<circle cx="25" cy="75" r="20"/>
```
> [Hi did I do it right?](https://github.com/EthanThatOneKid/acmcsuf.com/pull/672/commits/359c23a02aa36ea30a0baf54e815cd063d25e1d4)

You were close!

SVG files must begin with a root tag of `svg`.

```suggestion
<svg width="50" height="50" viewBox="0 0 50 50" fill="#2c91c6" xmlns="http://www.w3.org/2000/svg">
  <circle cx="25" cy="25" r="25"/>
</svg>
```

EthanThatOneKid reviewed (changes requested):
There is only a minor change keeping you from saying, "first try 😎", but luckily it is a super minor change needed to make your SVG file valid. I left a suggestion in my review, feel free to commit the suggestion directly or add it manually if you have additional changes :).

JHMuir committed:
Merge remote-tracking branch 'upstream/main' into jhm_branch

JHMuir committed:
Merge branch 'main' into jhm_branch

JHMuir committed:
Update circle-thing.svg

JHMuir committed:
Merge branch 'jhm_branch' of https://github.com/JHMuir/acmcsuf.com into jhm_branch

JHMuir committed:
Fixed circle-thing svg file

Renamed 'my-new-thing.ts' to 'circle-thing.ts'

Fixed the dimensions of 'circle-thing.svg'

EthanThatOneKid reviewed (approved):
Nice first contribution!
